3_control.md

```diff
@@ -11,6 +11,14 @@ Influencing factors for a stable system:
 How to reconnect a load after an overcurrent event?
 
 Suggestions:
-* Through intelligent communication and negotiation between the things using the energy (the drain) and the source of the energy?
-* By priority? - Drains with small consumption have higher priority? - So that light with higher priority comes before heavy equipment.
-* The source offers/bids free energy capacity by some sort of communication channel. The first drain that requests this free energy, partly or in whole, gets his share. Others follow according to some rules until the free portion of energy is reserved and in used. Drains that don't pick up their request will lose their share og the available energy. ?
+
+1.) Through intelligent communication and negotiation between the things using the energy (the drain) and the source of the energy?
+
+2.) By priority? - Drains with small consumption have higher priority? - So that light with higher priority comes before heavy equipment.
+
+3.) The source offers/bids free energy capacity by some sort of communication channel. The first drain that requests this free energy, partly or in whole, gets his share. Others follow according to some rules until the free portion of energy is reserved and in used. Drains that don't pick up their request will lose their share og the available energy. ?
+
+4.) Letting the output voltage grow liniarly as more current is available. Like with a certain factor between the two. Let's for instance say 12V equals 5 Amp, 24V equals 10 Amp, 36V equals 15 Amp, 48V equals at least 20 Amp - and in between those numbers linearly. On the drain side, there will something knowing this - and according to a priority start consuming. ?
```

Comment:
Yeah, so that's exactly one part of the droop control. Just that these values are not fixed, but can be individual per device. A low-power device supplying power to the grid would go down with its output voltage already at lower current compared to one which has high power reserves left.

The problem is that you can't tell how much power is available locally. So if an idling load is connected to the rest of the grid with a long and tiny wire, the voltage measured at the load would be quite high, but drop immediately if it starts drawing power. This could result in a hysteresis which is not easy to prevent.

Reply:
Yes. I see. Your right. I'll check the video out.